bundle-size/comment.js

```diff
@@ -4,14 +4,44 @@
 /** @typedef {import('./compareStats').DiffResult} DiffResult */
 /** @typedef {import('./compareStats').EntryDiffResult} EntryDiffResult */
 
-const prMessageSymbol = `<!-- grafana-plugin-actions-bundle-size-comment -->`;
 const prMessageUpdate =
   "Whenever this PR is updated, this comment will update to reflect the latest changes.";
-const prMessageWelcome = `### Bundle Size Changes
```

Comment:
In IRM monorepo we need different title for each pnpm workspace, see`getPrMessageTitle` function below